src/Components/BehaviorsModal/Internal/Widgets/ValueWidget/ValueWidget.styles.ts

```diff
@@ -65,6 +66,18 @@ export const getStyles = memoizeFunction((theme: Theme) =>
                 ...overflowStyles
             } as IStyle
         ],
+        expressionValueInvalidPlaceholder: [
```

Comment:
do we really need different styles for this rendering? It seems like it ought to be exactly the same as the value rendering?

Reply:
'--' was tiny with the FontSize.20 styles.  Created this style to make it more visible